mutiny-core/src/federation.rs

```diff
@@ -603,6 +603,29 @@ impl<S: MutinyStorage> FederationClient<S> {
         }
     }
 
+    pub(crate) async fn reissue(&self, oob_notes: OOBNotes) -> Result<bool, MutinyError> {
+        let logger = Arc::clone(&self.logger);
+
+        // Get the `MintClientModule`
+        let mint_module = self.fedimint_client.get_first_module::<MintClientModule>();
+
+        // TODO: (@leonardo) Do we need any `extra_meta` ?
+        // Reissue `OOBNotes`
+        let operation_id = mint_module.reissue_external_notes(oob_notes, ()).await?;
+
+        // TODO: (@leonardo) re-think about the results and errors that we need/want
+        match process_reissue_outcome(&mint_module, operation_id, logger.clone()).await? {
+            ReissueExternalNotesState::Created | ReissueExternalNotesState::Failed(_) => {
+                log_info!(logger, "re-issuance of OOBNotes failed!");
+                Err(MutinyError::FedimintReissueFailed)
+            }
+            _ => {
+                log_info!(logger, "re-issuance of OOBNotes was successful!");
+                Ok(true)
```

Comment:
This either returns an error or ok(true). Is there any other information we could return that'd be useful, otherwise should just return ()

Reply:
No, we don't. I'll update it to ()